client/src/app/router.coffee

```diff
@@ -0,0 +1,24 @@
+Marionette = require('backbone.marionette')
+SiteView = require('./views/site')
+ListSitesView = require('./views/list-sites')
+FormView = require('./views/form')
+HomeView = require('./views/home')
+
+module.exports = Marionette.AppRouter.extend(
+
+  routes:
```

Comment:
instead of using routes lets use appRoutes and set up a controller
https://github.com/samccone/edit.sx-frontend/blob/master/assets/js/_router.coffee